sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -311,7 +311,7 @@ Azure::Response<Models::Table> TableServiceClient::CreateTable(
     Core::Context const& context)
 {
   auto url = m_url;
-  url.AppendPath("Tables");
+  url.AppendPath(Azure::Core::Url::Encode("Tables"));
```

Comment:
Nit - no need to encode constant
```suggestion
  url.AppendPath("Tables");
```